### DISCUSSION TITLE
Error running node_exporter when SAN paths are managed by Veritas DMP

### DISCUSSION THREAD

**Discussion by yashnagar** [thorhs/node_exporter_aix#7] [General]
Hi there,
I am seeing issues running node exporter 1.12.1.0 on AIX 7.2  if we have disks managed by Veritas DMP.

root@host100[/]# /usr/local/bin/node_exporter_aix -p 10051
Node exporter for AIX version 1.12.1.0 listening on port 10051
Error calling perfstat_diskpath: Invalid argument
Error calling perfstat_diskpath: Invalid argument
Error calling perfstat_diskpath: Invalid argument
Error calling perfstat_diskpath: Invalid argument
Error calling perfstat_diskpath: Invalid argument
Error calling perfstat_diskpath: Invalid argument

is there any way I can disable disk path statistics monitoring?  please help.

**Comment by tamatam310** [comment]
installp -ac -d <package_file.bff>  i am unable to download .bff file

**Comment by yashnagar** [comment]
Try -
installp -aXd /mnt/prometheus/node_exporter/node_exporter_aix.1.15.7.0.bff
node_exporter_aix

you need to specify package name in the end.

**Reply by tamatam310** [reply]
Thanks for Giving reply
I requesting you, please provide     .bff file for me, This is easy for install package node_exporter in aix server. I strucked here only since six months.
